components/sync/driver/brave_sync_profile_sync_service.cc

```diff
@@ -23,10 +22,12 @@
 
 namespace syncer {
 
-BraveProfileSyncService::BraveProfileSyncService(InitParams init_params,
-                                                 Profile* profile)
+BraveProfileSyncService::BraveProfileSyncService(
+    InitParams init_params,
+    ProfileSyncServiceDelegate* profile_service_delegate)
```

Comment:
we can simply pass `DeviceInfoSyncService*` without the need of delegate.

Reply:
Thanks. Fixed at https://github.com/brave/brave-core/pull/6732/commits/c5a07eb7f57f7980b3edaa367ac48a33ba61b45f